types/artifacts.d.ts

```diff
@@ -789,6 +789,7 @@ declare global {
         layoutShiftMaxSessionGap1sLimit5s: number,
         layoutShiftMaxSliding1s: number,
         layoutShiftMaxSliding300ms: number,
+        newCumulativeLayoutShiftAllFrames: number,
```

Comment:
I didn't expect `new` to be part of the internal name :)

`layoutShiftMaxSessionGap1sLimit5sAllFrames`? :D

Reply:
`layoutShiftMaxSessionGap1sLimit5sAllFrames` is good